### ISSUE TITLE
az monitor scheduled-query create - ERROR: unrecognized arguments

### ISSUE THREAD

**Issue by dacla42** [Azure/azure-cli#18597]
this was working up until recently...

az monitor scheduled-query create -g rg -n appsvc-exception --scopes $appInsightsId --description "scheduled query rule alert" --condition "count 'query' > 0" --condition-query query="exceptions" --action $actionGroupId --target-resource-type microsoft.insights/scheduledqueryrules --location westeurope --evaluation-frequency 5m --window-size 15m --severity 0 --mute-actions-duration 5m --auto-mitigate false

```
"azure-cli": "2.25.0",
  "azure-cli-core": "2.25.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {
    "alertsmanagement": "0.1.0",
    "application-insights": "0.1.13",
    "scheduled-query": "0.2.2"
  }
```
  
`az : ERROR: unrecognized arguments: --condition-query query=requests --auto-mitigate false`

**Comment by yonzhan** [comment]
monitor

**Comment by kairu-ms** [comment]
Fixed in 0.3.1 please remove —auto-mitigate
